owlsim-services/src/main/java/org/monarchinitiative/owlsim/services/resources/OntologyMatchResource.java

```diff
@@ -61,6 +52,28 @@
         return matches;
     }
 
+    
     // TODO - API for comparing two entities
 
+    @GET
+    @Path("/{entity}/{ontology}")
+    @Timed
+    @CacheControl(maxAge = 2, maxAgeUnit = TimeUnit.HOURS)
+    @ApiOperation(value = "Match", response = MatchSet.class,
+    notes = "Additional notes on the match resource.")
+    public List<SimpleClassMatch> getEntityMatches(
+            @ApiParam(value = "entity, e.g. MP:0001951",
+            required = true) @PathParam("entity") String entity,
+            @ApiParam(value = "ontology to be matched, e.g. HP",
+            required = true) @PathParam("ontology") String ontology)
+                    throws UnknownFilterException, IncoherentStateException {
+    	
+    	// TODO it's weird to resolve the curie here
```

Comment:
Yep I was also leaning towards that solution, I'll make that change in my PR!

Thanks!